### PR TITLE
SSPS-571 Update DSP_ SSP_Coordination.md

### PR DESCRIPTION
**Pull request by mgrudziaz-rtbh**
Added alternative ways to attach "ae" field in bid requests.

### CODE REVIEW

**Comment by mgrudziaz-rtbh** [comment]
@piwanczak proszę o zerknięcie, czy jest ok.

File: DSP_ SSP_Coordination.md [hunk 1]
```diff
@@ -171,11 +172,38 @@ OpenRTB Core BidRequest.imp = {
 }
 ```
 
+Optionally we support also "ae" and "biddable" fields given in alternative sections of imp object. Like in imp -> ext -> igs
```
Comment (piwanczak) [inline DSP_ SSP_Coordination.md:175]: let's add a link to IAB specs that we reference

**Review by piwanczak — CHANGES_REQUESTED** [review]
please review the comments

File: DSP_ SSP_Coordination.md (hunk 1 above)
Comment (mgrudziaz-rtbh) [reply DSP_ SSP_Coordination.md:175]: Done

**Review by piwanczak — APPROVED** [review]

**Comment by mgrudziaz-rtbh** [comment]
@piwanczak nie mam możliwości merge'a. Mógłbyś zmergeować?

`This branch has no conflicts with the base branch
Only those with [write access](https://docs.github.com/articles/what-are-the-different-access-permissions) to this repository can merge pull requests.`

**Pull request merged**